comment/views/comments.py

```diff
@@ -120,23 +107,25 @@ def post(self, request, *args, **kwargs):
         return render(request, 'comment/comments/base.html', context)
 
 
-class ConfirmComment(View):
+class ConfirmComment(View, BaseCommentCreate):
     email_service = None
 
+    def _handle_invalid_comment(self, comment, request):
```

Comment:
Please make this method static.

Reply:
how would it use the email service then?

Reply:
The email service is not used inside this method!!

Reply:
I aksed you to make it static bcz `self` is not used at all.

Reply:
sorry, got confused with another comment.